docs/Agent/installation/ansible_based_installation.md

```diff
@@ -0,0 +1,37 @@
+# Ansible based installation
+
+In the following, we describe how to install Pbench Agent using an ansible playbook.
+
+:::{note}
+The same Pbench Agent version must be installed on all the test systems that participate in a benchmark run, there is no support for mixed installations.
+:::
+
+## Setup
+
+1. Make sure that you have ansible package installed.
```

Comment:
I think that Dave told you to remove the monospace font from "ansible", but now that it's in plain text it should probably be capitalized, since it's the name of a RH product.  And, it should probably have "the" added before it.